### PR TITLE
Feature/scene page

### PR DESCRIPTION
**Pull request by gfodor**
This adds a shiny new scene landing page.

### CODE REVIEW

**Comment by brianpeiris** [comment]
Broken in Chrome:
image

**Comment by brianpeiris** [comment]
Sometimes it seems to start the camera at the spawn point position instead of the screenshot position

File: src/react-components/scene-ui.js [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+              <button onClick={this.createRoom}>
+                <FormattedMessage id="scene.create_button" />
+              </button>
+            </div>
+          </div>
+          <div className={styles.info}>
+            <div className={styles.name}>{this.props.sceneName}</div>
+            <div className={styles.attribution}>{this.props.sceneAttribution}</div>
+          </div>
+          <div className={styles.description}>{this.props.sceneDescription}</div>
```
[the remark this answers is not in the document]
Comment (gfodor) [reply src/react-components/scene-ui.js]: Ah I want to leave it off for now, didn't realize it was still in the dom

File: src/scene.html [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+<html>
+
+<head>
+    <!-- DO NOT REMOVE/EDIT THIS COMMENT - META_TAGS -->
+
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
+
+    <link rel="shortcut icon" type="image/png" href="/favicon.ico">
+    <title>Scene | Hubs by Mozilla</title>
```
[the remark this answers is not in the document]
Comment (gfodor) [reply src/scene.html]: I have this working via reticulum now, so its crawlable

**Pull request merged**